vision/lidar/record_data.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/python3
+""" Record data from Ouster OS-1-32-U LiDAR sensor to pcap file."""
```

Comment:
Comment at top can be removed, unless this isn't being run in a poetry shell?

Reply:
The line there is not actually a comment but a shebang. It's there for convenience in making the file self-executable (runnable as `./record_data.py --args` instead of `python3 record_data.py --args`). This script isn't really meant to be used in the mission but instead as a tool for testing and debugging. Also, as I understand it, this doesn't stop it from running in a poetry shell nor does it change any of its behavior when running it there.
Would you still like me to remove this feature or would you rather keep it?

Reply:
Gotcha. No, I misunderstood what that was doing. You can keep that.